.gitignore

```diff
@@ -7,4 +7,7 @@
 
 terraform.tf
 aviator_pipeline.yml
+.project
```

Comment:
We generally don't add IDE-specific things to our `.gitignore` files; instead we prefer devs to configure their global gitignore settings so that they don't show up in any repos. This way, any dev can use any editor that they fancy and we don't have to adjust all of our repos to suit.

So...can you just delete these lines please :)